### DISCUSSION TITLE
Could a "loading bar" be added?

### DISCUSSION THREAD

**Discussion by TsukiZero** [ruffle-rs/ruffle#3003] [Ideas] [answered]
Flash Applications used to have those coded in, but Ruffle doesn't have any. Maybe we could have at least a basic "loading bar" with an option to show more details than just the bar and percentage, like telling the application's size and assets it's loading.

**Comment by ousia** [comment, marked as answer]
@TsukiZero, there are two issues dealing with this issue: #2302 and #875.

Maybe it would be better to discuss the issue there.

**Reply by nicomicap** [reply]
http://runouw.com/games/sm63/popup.php this one has one

**Comment by TsukiZero** [comment]
Alright, feel free to close this one then.

**Reply by ousia** [reply]
@TsukiZero, discussions aren’t closed, only they are marked as answered by their original author.
